index.js

```diff
@@ -104,6 +140,49 @@ class Client {
     }
     return analyses
   }
+
+  /**
+    * Runs MythX analysis return issue information and metadata regarding the run
+    *
+    * @param {options} object - structure which must contain:
+    *      {data} object       - information containing Smart Contract information to be analyzed
+    *      {timeout} number    - optional timeout value in milliseconds
+    *
+    * @returns object which contains:
+    *      (issues}  object - an like object which of issues is grouped by (file) input container.
+    *      {status}  object - status information as returned in each object of analyses().
+    *
+    **/
+  async analyzeWithStatus (options) {
+    const issues = await this.analyze(options)
+    const uuid = issues.uuid
+    delete issues.uuid
+    const status = await this.getStatus(uuid)
+    return {
+      issues,
+      status
+    }
+  }
+
+  async getStatus (uuid, inputApiUrl = defaultApiUrl) {
+    let accessToken = this.accessToken
+    if (!accessToken) {
+      const tokens = await login.do(this.email, this.ethAddress, this.password, this.apiUrl)
+      accessToken = tokens.access
+    }
+    const url = `${inputApiUrl}/${defaultApiVersion}/analyses/${uuid}`
+    return simpleRequester.do({ url, accessToken: accessToken, json: true })
+  }
+
+  async getIssues (uuid, inputApiUrl = defaultApiUrl) {
+    let accessToken = this.accessToken
+    if (!accessToken) {
+      const tokens = await login.do(this.email, this.ethAddress, this.password, this.apiUrl)
+      accessToken = tokens.access
+    }
+    const url = `${inputApiUrl}/${defaultApiVersion}/analyses/${uuid}/issues`
+    return simpleRequester.do({ url, accessToken: accessToken, json: true })
```

Comment:
If you still want to keep the local `accessToken` you can take advantage of shorthand property names when initializing the object literal:
```suggestion
    return simpleRequester.do({ url, accessToken, json: true })
```

Reply:
Don't want the local accessToken. That was an oversite that I am glad you pointed out. Thanks.